src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -109,12 +109,27 @@ This component allows the user to pick an emoji.
 			:show-preview="showPreview"
 			:title="previewFallbackTitle"
 			v-bind="$attrs"
-			@select="select" />
+			@select="select">
+			<template #searchTemplate="slotProps">
+				<NcTextField ref="search"
+					class="search"
+					:value.sync="search"
+					:label="t('Search')"
+					:label-visible="true"
+					:placeholder="i18n.search"
+					trailing-button-icon="close"
+					:trailing-button-label="t('Clear search')"
+					:show-trailing-button="search !== ''"
+					@trailing-button-click="clearSearch(); slotProps.onSearch(search);"
+					@update:value="slotProps.onSearch(search)" />
```

Comment:
Shall we maybe also show a `Magnify` icon to indicate the field is for searching?
```suggestion
					@update:value="slotProps.onSearch(search)">
					<Magnify :size="16" />
				</NcTextField>
```

Reply:
Was considered, though the other search fields, e.g. unified search and contacts search, do not have a magnify icon so will keep it consistent here :)